src/BuildIt.Forms/BuildIt.Forms.Controls/Enums.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace BuildIt.Forms.Controls
+{
+    /// <summary>
+    ///  The available focus modes
+    /// </summary>
+    public enum CameraFocusMode
+    {
+        // Default value
+        None,
+
+        // Auto-focus
```

Comment:
lol - comments not helpful - please describe what the difference is between Auto-focus and continuous auto-focus